fidoutils.js

```diff
@@ -276,6 +276,15 @@ function canAuthenticateWithCredId(options) {
 	return canAuthenticate;
 }
 
+function getFidoUtilsConfigfromNodeClient() {
```

Comment:
I have to ask - why is this function called `getFidoUtilsConfigfromNodeClient` and not just `getFidoUtilsConfig` ?

If you are going to have settings and getters, they really should be symmetric (setXXX, getXXX).